src/app/assessment/assessment.component.html

```diff
@@ -27,7 +27,19 @@ <h1>{{ assessment.name }}</h1>
     </p>
   </div>
 
-  <form [formGroup]="questionsForm">
+  <ion-list *ngIf="submission.isLock" class="member-detail-container no-bg" lines="none">
+    <ion-item lines="none">
+      <ion-avatar slot="start">
+        <img [src]="submission.image ? submission.image : 'https://my.practera.com/img/user-512.png'">
+      </ion-avatar>
+      <ion-label>
+        <h4>Locked by {{ submission.submitterName }}</h4>
+        <p>Please wait until the user finishes editing</p>
+      </ion-label>
+    </ion-item>
+  </ion-list>
+
+  <form [ngClass]="{'readonly-assessment' : submission.isLock}" [formGroup]="questionsForm">
```

Comment:
This is not how we show "readonly"
readonly should be the same as when student see the submission